### ISSUE TITLE
Twisted Website design

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8833]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| gravis reported|
|-|-|
|Trac ID|trac#407|
|Type|enhancement|
|Created|2003-11-19 22:38:09Z|

<details><summary>Searchable metadata</summary>

```
trac-id__407 407
type__enhancement enhancement
reporter__gravis gravis
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__ 
time__1069281489000000 1069281489000000
changetime__1469431178923145 1469431178923145
version__ 
owner__ 
cc__radix cc__spiv cc__gravis cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
some pages like :
http://www.twistedmatrix.com/documents/howto/woven
are really graphically poor. No images, no colors, big fonts, etc. Moreover, the
site should have a navbar on the top with a twisted logo.

I can help to redisign to site if so (radix ?) wants to.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
just wanted to add that I love the plone style on zope.org. :)
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Why is no images and no colours a *bad* thing? :)

Also, the fonts are NOT big.  They are normal size; adjust your web browser if
its default font size is too big for your tastes.

It might be worth adding a simple navbar to the howtos, the single "Index" link
at the bottom is pretty obscure.  Feel free to propose patches to the template
(and the stylesheet if you think you can make it prettier).
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
Where can I find the templates sources ? in twisted archive ? no images and no
colours are "bad" because the site seems a little unfinished :)
I'll provide some patches
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
The template used for howtos is in doc/howto/template.tpl
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
Actually, remember, we have a different one for the website: it's
doc/howto/website-template.tpl

I don't remember why it's different, though :)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
as nothing ever came of this issue, I'm resolving it.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_9"></a>|GitHub commented|
|-|-|

In changeset af87497e6bcd6bf1c59293399e15a7fb8997fe5b
```
#!CommitTicketReference repository="" revision="af87497e6bcd6bf1c59293399e15a7fb8997fe5b"
Merge pull request #407 from twisted/8667-rodrigc-banana-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8667

Port twisted.spread.banana to Python 3
```
